cli/src/build/onboarding/ui/app.tsx

```
@@ -148,6 +200,26 @@ const OnboardingApp: FC<AppProps> = ({ appId, initialProgress, iosDir, apikey })
   const [ciSecretExistingKeys, setCiSecretExistingKeys] = useState<string[]>([])
   const [ciSecretError, setCiSecretError] = useState<string | null>(null)
   const [ciSecretUploadSummary, setCiSecretUploadSummary] = useState<string | null>(null)
+  // GitHub Actions workflow setup state. setupMode tracks the 3-way choice at
+  // ask-github-actions-setup. After a successful secrets upload, with-workflow
+  // continues into pick-build-script + writing-workflow-file; secrets-only
+  // exits via build-complete; declined branches to ask-export-env.
+  const [setupMode, setSetupMode] = useState<'undecided' | 'with-workflow' | 'secrets-only' | 'declined'>('undecided')
+  const [availableScripts, setAvailableScripts] = useState<Record<string, string>>({})
+  const [recommendedScript, setRecommendedScript] = useState<string | null>(null)
+  const [buildScriptChoice, setBuildScriptChoice] = useState<BuildScriptChoice | null>(null)
+  const [pendingCustomCommand, setPendingCustomCommand] = useState<string>('')
```

ESLint

[error] 'pendingCustomCommand' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)